python/langsmith/client.py

```diff
@@ -837,6 +896,7 @@ def batch_ingest_runs(
         update: Optional[
             Sequence[Union[ls_schemas.Run, ls_schemas.RunLikeDict, Dict]]
         ] = None,
+        pre_sampled: bool = False,
```

Comment:
We should document this - not super obvious what the effect is just by looking at the signature.

And should this be in the public API anyways? We could put the shared logic in a private method and then call that from our queue

Reply:
I've documented, left as public as could be useful to someone else conceivably